src/RVRP_solution.jl

```diff
@@ -16,13 +16,18 @@ struct Route
     # path::Path
 end
 
+struct RVRPfinalState # received as the target end of period situation
+    id::String
+    ongoing_routes::Vector{Route} # holds the backlog of uncompleted requests
+    uncompleted_request_ids::Vector{String} # the request is partially fulfilled 
+end
+
+
 struct RvrpSolution
     id::String
     instance_id::String
     cost::Float64
     routes::Vector{Route}
-    unassigned_pickup_ids::Vector{String}
-    unassigned_delivery_ids::Vector{String}
-    unassigned_shipment_ids::Vector{String}
-    uncompleted_shipment_ids::Vector{String} # the request is partially fulfilled 
+    unassigned_request_ids::Vector{String}
+    final_state::RVRPfinalState  # the request is partially fulfilled 
```

Comment:
The final_state is only here so I guess it s ok to put the whole structure and not the id. Right?